app/views/menu/index.html.erb

```diff
@@ -5,7 +5,7 @@
       <%= item.name %>
       <%= button_to "Add to Cart", cart_items_path(item: item), class: "btn btn-default" %>
       <div class="btn-group btn-group-xs" role="group">
-      <% item.categories.each do |item_category| %>
+      <% item.categories.uniq.each do |item_category| %>
```

Comment:
I'm not sure I understand why there should be a uniq here? It seems like it might be conceptually better to disallow an admin from tagging a single item with the same category multiple times, no?